Phonebook.Frontend/src/app/modules/rooms/components/room/room.component.ts

```diff
@@ -55,7 +55,7 @@ export class RoomDetailComponent implements OnInit {
             this.room = this.node.data as Room;
           }
           this.personService
-            .getPersonsByRoom(RoomHelpers.getParamsAsArray(params, ['cityId', 'buildingId', 'floorId', 'roomId']))
+            .getPersonsByRoom(RoomHelpers.getParamsAsArray(params, ['roomId'])[0])
```

Comment:
Das funktioniert nicht, es gibt Räume die die gleiche RoomId haben, oder mittlerweile nicht mehr?

Reply:
test before close. The room Id is now the DatabaseId, thats mean it is unique.